SingularityService/src/main/java/com/hubspot/singularity/data/SingularityValidator.java

```diff
@@ -32,6 +33,7 @@
 import com.hubspot.singularity.WebExceptions;
 import com.hubspot.singularity.config.SingularityConfiguration;
 import com.hubspot.singularity.data.history.DeployHistoryHelper;
+import org.dmfs.rfc5545.recur.RecurrenceRule;
```

Comment:
can you sort these imports? if you're using Eclipse or IntelliJ, we have a formatter config here: https://github.com/HubSpot/Singularity/blob/master/eclipse/singularity_eclipse_formatter.xml